versions/3.0.4.md

```diff
@@ -2409,22 +2411,22 @@ This mechanism is used by [Link Objects](#link-object) and [Callback Objects](#c
 The runtime expression is defined by the following [ABNF](https://tools.ietf.org/html/rfc5234) syntax
 
 ```abnf
-    expression = ( "$url" / "$method" / "$statusCode" / "$request." source / "$response." source )
-    source = ( header-reference / query-reference / path-reference / body-reference )
+    expression = "$url" / "$method" / "$statusCode" / "$request." source / "$response." source
```

Comment:
Are we fine with the fact that ABNF literal text strings enclosed in double-quotes are case-insensitive?

For example `"$url"` matches values `$uRl`, `$url`, `$URL` and any other case variation.

Do we assume this to be general knowledge, or should we call it out?